### ISSUE TITLE
Mesure de visibilité (Search Console)

### ISSUE THREAD

**Issue by christianroy** [a10s-ca/referentiel#8]
Configurer Google Search Console (prérequis: nom de domaine)

**Comment by v-cao** [comment]
On pourrait se trouver un moment pour le faire :).

**Comment by christianroy** [comment]
> On pourrait se trouver un moment pour le faire :).

oui, je prévois démarrer ça bientôt et te solliciterai à l'étape de l'entrée DNS!

**Comment by christianroy** [comment]
> > On pourrait se trouver un moment pour le faire :).
> 
> oui, je prévois démarrer ça bientôt et te solliciterai à l'étape de l'entrée DNS!

@v-cao je viens de voir qu'il y a encore plus simple puisque GoDaddy est utilisé... Tu peux aller activer Search Console directement puisque tu as accès à GoDaddy. Les étapes sont les suivantes:

* tu vas sur https://search.google.com/search-console/
* dans le menu en haut à gauche pour sélectionner une propriété, tu vas à la fin et choisis "Ajouter une propriété"
* dans la fenêtre suivante:
 
![image](https://github.com/user-attachments/assets/afe1f1ff-52fd-4141-8fd2-99b461e21ec2)

* tu écris "datascene.ca" dans l'option Domaine, puis cliques sur "Continuer"
* tu suis les instructions (qui impliqueront de se connecter à GoDaddy)

Une fois la propriété validée, dans le menu de gauche il y a Paramètres, puis dans Utilisateurs et autorisation, il y a un bouton pour ajouter des utilisateurs, tu pourras m'ajouter (et les autres membres du comité).

**Comment by v-cao** [comment]
C'est fait !

**Comment by christianroy** [comment]
> C'est fait !

super! Une fois que ce sera complété, tu pourras nous partager les accèsd!

**Comment by christianroy** [comment]
> > C'est fait !
> 
> super! Une fois que ce sera complété, tu pourras nous partager les accès!

@v-cao je viens de repenser à ceci! Si la propriété est active et si tu en as la possibilité, ce serait bien de nous donner accès (ou à au moins un.e collègue chez Synapse C) avant vendredi!
